nautobot/ipam/tests/test_models.py

```diff
@@ -495,48 +495,6 @@ def test_duplicate_nonunique_role(self):
             role=roles[1],
         )
 
-    def test_multiple_nat_outside(self):
```

Comment:
Should some part of this test be kept to verify that `nat_outside_list` works as expected? Or is that already adequately tested elsewhere?